packages/squiggle-lang/src/rescript/Distributions/PointSetDist/PointSetDist_Scoring.res

```diff
@@ -0,0 +1,7 @@
+module LogScoring = {
+  let logFn = Js.Math.log2
+  let subtraction = (a, b) => Ok(a -. b)
+  let logScore = (a: float, b: float): result<float, Operation.Error.t> => Ok(
```

Comment:
since this is a `result` shouldn't it check if `a` is zero, if the quotient is below zero, or if `b` is zero?

Reply:
it should, you're right

Reply:
Flag to do this